packages/server/src/app.ts

```diff
@@ -145,6 +149,35 @@ export async function initApp(app: Express, config: MedplumServerConfig): Promis
   app.use(attachRequestContext);
   app.use(getRateLimiter(config));
   app.use('/fhir/R4/Binary', binaryRouter);
+
+  // Handle async batch by enqueueing job
+  app.post(
```

Comment:
Can we move this handler to separate file?  I feel like `app.ts` is already too big, and individual handlers are a pretty clean way to keep things organized.